sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/CreateBatchOptions.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus.implementation;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * The set of options that can be specified when creating an batch of messages. This wrapper will help to limit
+ * the messages with maximum allowed size.
+ *
+
+ */
+@Fluent
+public class CreateBatchOptions {
```

Comment:
The "MessageBatch" is not publicly exposed. We haven't discussed having this made public.

Reply:
The sendBatch APIs don't expose any of these options.

https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.imessagesender.sendbatch?view=azure-java-stable#com_microsoft_azure_servicebus_IMessageSender_sendBatch_Collection___extends_IMessage__

Reply:
Why does this exist??